### PR TITLE
[Finishes #187354184] Set up the frontend repository

### PR DESCRIPTION
**Pull request by Favor-star**
## What has been done so far?
In this PR, I have managed to set up the the react front end repository using `vite` command: `npm create vite@latest` as well as setting up the `tailwind css`.
*All of the files were added for the first time, and tailwind css is working as intended.*

### CODE REVIEW

**Review by hozayves — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Vite + React + TS</title>
+  </head>
+  <body>
+    <div id="root"></div>
+    <script type="module" src="/src/main.tsx"></script>
+  </body>
+</html>
```
Comment (niyontwali) [inline index.html]: Please refine this file to include the following 

1. Usefully meta data for SEOs
2. Include a favicon
3. Include a proper title for our e-commerce

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  }
+}
+
+.card {
+  padding: 2em;
+}
+
+.read-the-docs {
+  color: #888;
+}
```
Comment (niyontwali) [inline src/App.css]: Please remove this whole file we shall use the index.css and also delete the global.css.

File: src/assets/styles/styles.css [hunk 3]
```diff
@@ -0,0 +1,4 @@
+/* Grobal css */
+body {
+  box-sizing: border-box;
+}
```
Comment (niyontwali) [inline src/assets/styles/styles.css]: Remove this file and its folder

**Review by niyontwali — CHANGES_REQUESTED** [review]
Please work on the feedback provided on specific files and as the below'

1. Create a PR template.
2. Implement Husky.
3. Implement ESLint and Prettier.
4. Delete the styles, config, and context folders.

Otherwise the rest look better

**Review by Mag-codes — APPROVED** [review]
LGTM

**Review by JeanIrad — APPROVED** [review]

**Review by niyontwali — APPROVED** [review]

**Review by niyontwali — APPROVED** [review]

**Pull request merged**